src/features/defi/helpers/normalizeOpportunity.tsx

```diff
@@ -70,16 +70,19 @@ type NormalizeOpportunitiesProps = {
   cosmosSdkStakingOpportunities: MergedActiveStakingOpportunity[]
   foxEthLpOpportunity?: EarnOpportunityType
   stakingOpportunities?: EarnOpportunityType[]
+  lpOpportunities?: EarnOpportunityType[]
 }
 
 export const useNormalizeOpportunities = ({
   cosmosSdkStakingOpportunities = [],
   foxEthLpOpportunity,
   stakingOpportunities,
+  lpOpportunities,
 }: NormalizeOpportunitiesProps): EarnOpportunityType[] => {
   return [
     ...useTransformCosmosStaking(cosmosSdkStakingOpportunities),
     ...(stakingOpportunities ? stakingOpportunities : []),
     ...(foxEthLpOpportunity ? [foxEthLpOpportunity] : []),
+    ...(lpOpportunities ? lpOpportunities : []),
```

Comment:
Very nice catch selecting all LP opportunities instead of passing the ETH/FOX one only, which was a legacy from previous implementation.

Wondering since we will bail out from resolution with the flag off, would it make sense to always pass `lpOpportunities` only so we can remove the line above and l.78?

Reply:
Ah, good idea! That will also allow me to delete a few lines in `AllEarnOpportunities.tsx`.

I'll make that change now.